roles/kubernetes/node/templates/kubelet.j2

```diff
@@ -12,6 +12,13 @@ KUBELET_ADDRESS="--address={{ ip | default("0.0.0.0") }}"
 # KUBELET_PORT="--port=10250"
 # You may leave this blank to use the actual hostname
 KUBELET_HOSTNAME="--hostname-override={{ ansible_hostname }}"
+{% if inventory_hostname in groups['kube-master'] and inventory_hostname not in groups['kube-node'] %}
```

Comment:
I see 3 cases:
1. A host is only a master (standalone_kubelet=true) and admin does not want to have the master as part of the cluster (register_master is false).
2. A host is only a master (standalone_kubelet=true) and admin does want to have the master as part of the cluster (register_master is true).
3. A host is a master and node at the same time (standalone_kubelet=false). In this case register_master does not matter, as the node MUST be registered